sdk/servicebus/azure-servicebus/samples/sync_samples/connection_string_parser.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+An example to show the usage of connection string parser.
+"""
+
+import os
+from azure.servicebus import (
+    ServiceBusClient,
+    ServiceBusMessage,
+    ServiceBusConnectionStringProperties,
+    parse_connection_string,
+)
+
+conn_str = os.environ['SERVICE_BUS_CONN_STR']
+QUEUE_NAME = os.environ["SERVICE_BUS_QUEUE_NAME"]
+
+parse_result = parse_connection_string(conn_str)
+
+fully_qualified_namespace = parse_result.fully_qualified_namespace
+print(fully_qualified_namespace)
+
+# the name of the queue/topic
+entity_path = parse_result.entity_path
+
+# initialize with DefaultAzureCredential
+servicebus_client = ServiceBusClient(
+    fully_qualified_namespace,
+    DefaultAzureCredential(),
+    entity_path
```

Comment:
as per https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py#L76-L77

`entity_name` is a keyword only argument and something we don't want to promote.
besides, I find it is inconsistent with the name `entity_path`.

tbh, I'm fine if we don't have a sample code of parse for now otherwise I would just do parse and print properties stuff depending on it's so difficult to find a proper example to utilize the property bags.

Reply:
removing the example now. I think let's change to `entity_name` for  consistency